stages/text-extraction/src/main/java/edu/kit/kastel/mcse/ardoco/core/textextraction/informants/MappingCombinerInformant.java

```diff
@@ -41,6 +42,9 @@ private void combineSimilarPhraseMappings(TextState textState) {
         for (PhraseMapping phraseMapping : phraseMappings) {
             ImmutableList<PhraseMapping> similarPhraseMappings = phraseMappings.select(p -> SimilarityUtils.getPhraseMappingSimilarity(textState, phraseMapping,
                     p, PhraseMappingAggregatorStrategy.MAX_SIMILARITY) > minCosineSimilarity);
+            // Remove the phrase mapping from the list of similar phrase mappings
+            // TODO This breaks the logic but seems to be logical ..
```

Comment:
Is this TODO and the commented code up to date? Can it be removed?

Reply:
It's a finding of mine. I'd not remove that comment.